modules/hostap/Kconfig

```diff
@@ -109,9 +109,14 @@ config WIFI_NM_WPA_SUPPLICANT_WEP
 
 choice WIFI_NM_WPA_SUPPLICANT_CRYPTO_BACKEND
 	prompt "WPA supplicant crypto implementation"
+<<<<<<< HEAD
```

Comment:
merge conflict?

Reply:
Jumped the gun there. Saw that this is resolved in next commit. Can we squash them